docs/multisig-guide.md

```diff
@@ -0,0 +1,115 @@
+
+* Seed backups (on steel) are not protected by a pin as the *signing device*s (e.g. *hardware wallets*). They therefore need a **secure location**, i.e. accessible only to you or people you completely trust.
+  * Examples are: Second home/apartment, Safe deposit box, a relatives (or very good friends) house/apartment
+* In 1 location there should not be more than 1 seed
+  * Minimum is: 3 **secure locations**, in each location is the signing device and its seed backup
+  * Best is: 3 **secure locations** for the seed backups **and** 3 (medium) secure locations for the *signing device*s (e.g. *hardware wallets*).
+* Please read: [https://bitcoiner.guide/multisig/backup]( https://bitcoiner.guide/multisig/backup)
+
+
+
+### 2. Multisig wallet creation with Specter Desktop
+
+From each seed the *signing device*  (e.g. *hardware wallet*)  will calculate a *(master) public key* and the *fingerprint* . These can be exported (via USB, QR Code, or sd-card) to Specter Desktop:
+
+![xpubs](images/multisig-guide/xpubs.svg)
+
+* Specter Desktop will take the  *(master) public key*s together with the *derivation paths* (for segwit multisig the standard is "m/48h/1h/0h/2h") to construct the (watch-only) wallet. The *fingerprints* are important, when then *signing devices* sign transactions.
+* **Keep a copy of all *(master) public keys* and *fingerprints* (i.e. the Specter "Printable PDF backup") with every seed backup**.  They allow anyone to recreate the (watch-only) wallet and see the Bitcoin balance, and therefore should be kept **private**.   
+
+
+
+### 3. Register the multisig setup with each *signing device*  (e.g. *hardware wallet*)
+
+*Signing devices* (see [here](multisig-security-tradeoffs.md) for a comparison) can import a multisig-wallet file (different ways to do this for every vendor, [coldcard](https://bitcoiner.guide/multisig/wallet/#notify-coldcard), [cobo/keystone](https://bitcoiner.guide/multisig/wallet/#notify-cobo), [bitbox02](https://shiftcrypto.ch/blog/specter-multisig-guide/)). 
+
+* Then the *signing device* knows it's parts of a multisig setup and can protect against various attacks.
+
+
+
+### 4 Testing all parts of the setup
+
+Because the multisig setup is more complicated, testing all parts of the setup is even more important.  All the tests should be done only with **minimal funds** on the multisig wallet.
+
+#### 4a Test the seed backups
+
+Crucial is that you backed up each seed correctly (on steel). Examples:
+
+* Double and triple checking by comparing the steel backup with the *signing device*  (also known as *hardware wallet*)
+* a) Wiping the *signing device*  (also known as *hardware wallet*), b) restoring the  *signing device*  from the backup, c) exporting the  *(master) public key*  ensuring it is the same as before.
+
+#### 4b (Watch-only) Wallet test
```

Comment:
I think the pennytest shouldn't be mentioned last. Imho, it should even be combined with the seed-backup-test:
* After initial setup, send a small amount in
* wipe the setup
* recreate the setup from the backups and send the money out again

Reply:
It's not yet changed?!

Reply:
What do you mean? I changed section 4.1 through 4.4.

Reply:
Did you forget to push? the pennytest is still mentioned last and the seed-backuptest in 4a does not mention the pennytest, yet?

Reply:
Just to be sure, I merged the latest master into the branch https://github.com/cryptoadvance/specter-desktop/pull/1731/commits/e4330f8227b3e16bb6e93056fe9ffedb290ddfee and the https://deploy-preview-1731--specter-desktop-docs.netlify.app/multisig-guide/ looks like
![image](https://user-images.githubusercontent.com/60378539/170695350-43827ef1-1ee7-4198-ab2b-fcf92b2b4db5.png)

I am unsure which commit you are referring to.

Reply:
Nono, all good! I clicked on the "View changes" button on the top of this comment and i have no idea what's the purpose of that button if it doesn't show the recent changes.